From cb4acfa372a8d66703706ef54f985bd588d8bb99 Mon Sep 17 00:00:00 2001
From: Andre Paiusco <github@paiusco.org>
Date: Sat, 12 Oct 2024 16:29:01 +0200
Subject: [PATCH 198/201] mdadm.man: Add udev-rules flag

--udev-rules flag is added and point to mdadm.conf man page
for further explanations about POLICY.

Signed-off-by: Andre Paiusco <github@paiusco.org>
---
 mdadm.8.in | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/mdadm.8.in b/mdadm.8.in
index ee5b9bd4..43d6fe05 100644
--- a/mdadm.8.in
+++ b/mdadm.8.in
@@ -1830,6 +1830,16 @@ can be found it
 under
 .BR "SCRUBBING AND MISMATCHES" .
 
+.TP
+.B \-\-udev\-rules=
+it generates the udev rules to the file that handles hot-plug bare devices.
+Given the POLICYs defined under
+.IR {CONFFILE}\ (or {CONFFILE2})
+
+See
+.BR mdadm.conf (5)
+for more details and usage examples about POLICY.
+
 .SH For Incremental Assembly mode:
 .TP
 .BR \-\-rebuild\-map ", " \-r
-- 
2.41.0

